src/explore-education-statistics-admin/src/pages/publication/components/PublicationForm.tsx

```diff
@@ -152,30 +68,13 @@ const PublicationForm = ({
     }
 
     return schema;
-  }, [initialValues?.topicId, approvedMethodologies]);
+  }, [initialValues?.topicId]);
 
-  const handleSubmit = useFormSubmit(
-    async ({
-      methodologyChoice,
-      methodologyId,
-      externalMethodology,
-      ...values
-    }: FormValues) => {
-      switch (methodologyChoice) {
-        case 'existing':
-          await onSubmit({ ...values, methodologyId });
-          break;
-        case 'external':
-          await onSubmit({ ...values, externalMethodology });
-          break;
-        default:
-          await onSubmit(values);
-      }
-    },
-    errorMappings,
-  );
+  const handleSubmit = useFormSubmit(async ({ ...values }: FormValues) => {
```

Comment:
Do you need the object spreading here?  Or could it just be:

`useFormSubmit(async (values: FormValues) =>`